README.md

```diff
@@ -1 +1,80 @@
-# New-repo
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>New repo</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [👥 Authors](Sohail-Aslam)
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+# 📖 [New repo] <a name="about-project"></a>
+
+**[New repo]** is a  analyze code and provide feedback on potential issues, violations of coding standards, and common programming mistakes.
+
+
+
+### Key Features <a name="Error detection"></a>
+Error detection is used in software development to analyze source code and identify potential issues or errors. Linters are commonly used in various programming languages to enforce coding standards, improve code quality, and detect bugs early in the development process.
+
+- **[Automated Checks]**
+- **[Code Quality Analysis]**
+- **[Error detection]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="Sohail-Aslam"></a>
+
+👤 **Sohail-Aslam**
+
+- GitHub: [@githubhandle](https://github.com/Sohail-Aslam/New-repo)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/feed/)
+
```

Comment:
- [ OPTIONAL] Next time use your username in the [] brackets in the Author's Section @Sohail-Aslam